### PR TITLE
App Services - Modules

### PR DESCRIPTION
**Pull request by johnpapa**
#sign-off

### CODE REVIEW

File: docs-ref-services/appservice.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ App Service includes the web and mobile capabilities that we previously delivere
 
 ## Management Package
 
-### Install npm package
+### Install the npm package
 
 Install the Azure storage module for Node.js
```
Comment (clarkio) [inline docs-ref-services/appservice.md]: Text needs to change to `Azure App Service` module

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/appservice.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-22/docs-ref-services/appservice.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180428203453-125%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
fixed. LGTM

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/appservice.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-22/docs-ref-services/appservice.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180442262054-125%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@johnpapa merged

**Pull request merged**